docs/walkthrough/walkthrough-kind.md

```diff
@@ -69,6 +69,13 @@ CURRENT   NAME                                                 CLUSTER
 
 The `kind-walkthrough` is the context for the cluster you created and the `*` at the beginning means that it is the currently active context.
 
+If you do not see `*` for `kind-walkthrough` that means its not currently active you can run 
+below command to select it and make it as active cluster 
```

Comment:
We've already explained in the previous line that the `*` indicates the active context so no need to repeat that here, we can simplify it:

```suggestion
If it's not currently active you can run the following command to select it as the active cluster:
```